### PR TITLE
Logo change

### PR DESCRIPTION
**Pull request by stephencookdev**
# Problem
Our footer's logo is the old logo

# Solution
Update the logo to the new one 🙂 

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [n/a] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-550-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://550-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-550-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/assets/powered-by-onfido.svg [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+    <path d="M98.8 11.58c-1.19 0-2.08.48-2.6 1.4V11.8h-1.36v8.14h1.36V15.7c0-.86.21-1.55.63-2.06.43-.5 1-.76 1.75-.76.62 0 1.12.2 1.48.6.36.41.54 1.04.54 1.86v4.58h1.37V15.1c0-1.16-.28-2.06-.83-2.64a3.17 3.17 0 0 0-2.34-.87zM126.21 11.58a4.28 4.28 0 0 0-4.25 4.29c0 2.36 1.91 4.3 4.25 4.3a4.28 4.28 0 0 0 4.25-4.3c0-2.36-1.9-4.3-4.25-4.3zm2.87 4.29c0 1.65-1.29 3-2.87 3a2.93 2.93 0 0 1-2.86-3c0-1.65 1.29-3 2.86-3a2.93 2.93 0 0 1 2.87 3zM89.29 11.58a4.28 4.28 0 0 0-4.25 4.29c0 2.36 1.9 4.3 4.25 4.3a4.28 4.28 0 0 0 4.25-4.3c0-2.36-1.9-4.3-4.25-4.3zm2.86 4.29c0 1.65-1.28 3-2.86 3a2.93 2.93 0 0 1-2.86-3c0-1.65 1.29-3 2.86-3a2.93 2.93 0 0 1 2.86 3zM119.37 12.94a4 4 0 0 0-2.98-1.36 4.2 4.2 0 0 0-4.1 4.29 4.2 4.2 0 0 0 4.1 4.3 4 4 0 0 0 2.98-1.37v1.14h1.37V8.5h-1.37v4.44zm-.04 2.93c0 1.65-1.28 3-2.86 3a2.93 2.93 0 0 1-2.86-3c0-1.65 1.29-3 2.86-3a2.93 2.93 0 0 1 2.86 3zM110.26 8.12a1 1 0 0 0-.74.3c-.2.2-.3.45-.3.75 0 .28.1.53.3.74.2.2.45.3.74.3a1 1 0 0 0 .73-.3c.2-.2.3-.45.3-.74 0-.3-.1-.54-.3-.75-.2-.2-.45-.3-.73-.3zM109.58 11.8h1.36v8.14h-1.36zM105.7 11.8v-1.6c0-.4.31-.73.71-.73h1.6V8.22h-1.75c-1.06 0-1.93.87-1.93 1.95v1.64h-1.5v1.27h1.5v6.86h1.37v-6.86h2.3V11.8h-2.3z"/>
```
Comment (stefaniacardenas) [inline src/components/Theme/assets/powered-by-onfido.svg]: Are these SVGs optimised?

Comment (stephencookdev) [reply src/components/Theme/assets/powered-by-onfido.svg]: They are - I think they're just hard paths to optimise :/ But if I try and optimise any further, you can visually see the SVG quality degrade

**Pull request merged**